### PR TITLE
fix(gulp): replacing gulp-cssmin which has been deprecated and un-maintained with gulp-csso

### LINKED ISSUE #1251: Replace gulp-cssmin
Package `gulp-cssmin` at https://github.com/chilijung/gulp-cssmin seems un-maintained and old while other candidates are:
1. gulp-clean-css at https://www.npmjs.com/package/gulp-clean-css
2. gulp-csso at https://www.npmjs.com/package/gulp-csso/

### PR DESCRIPTION
**Pull request by lirantal**
Fixes #1251 - see the issue for motivation and other details

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -91,7 +91,7 @@
     "gulp-autoprefixer": "~3.1.0",
     "gulp-concat": "~2.6.0",
     "gulp-csslint": "~0.2.0",
-    "gulp-cssmin": "~0.1.7",
+    "gulp-csso": "^1.1.0",
```
Comment (ilanbiala) [inline package.json]: ~

**Comment by mleanos** [comment]
LGTM

**Comment by simison** [comment]
Some interesting discussion regarding this stuff:
https://github.com/gulpjs/plugins/issues/210
https://github.com/plus3network/gulp-less/pull/221

>the current flow by the gulp team is to use: https://github.com/postcss/gulp-postcss with a postcssnano plugin, or https://github.com/scniro/gulp-clean-css

**Comment by codydaig** [comment]
LGTM

**Comment by ilanbiala** [comment]
LGTM.

**Pull request merged**